pages/apim/3.x/kubernetes/apim-kubernetes-operator-installation.adoc

```diff
@@ -0,0 +1,41 @@
+[[apim-kubernetes-operator-installation]]
+= GKO installation and deployment
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_kubernetes_operator_installation.html
+:page-folder: apim/kubernetes
+:page-layout: apim3x
+
+You can deploy the Gravitee Kubernetes Operator [GKO] in two ways:
```

Comment:
IMHO it's not really 2 ways to deploy. 
more something like : 
If you have an existing cluster (with api management & gateway ) then you can read this section. 
Otherwise you can do this part to set up a local cluster.

Reply:
we will talk later about several ways to deploy / install GKO 

we plan to have 3 way to do this in the future : 
- the one by bundle.yml
- install with OLM and operatorhub  
- install with helm

Reply:
Agree - updated the docs